airflow/gcp/operators/pubsub.py

```diff
@@ -572,6 +588,10 @@ def execute(self, context):
 class PubSubPublishOperator(BaseOperator):
     """Publish messages to a PubSub topic.
 
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
```

Comment:
I am not sure about the comma, @mschickensoup WDYT?

Reply:
This sentence is used widely with comma 🤔 . Is it proper sentence @mschickensoup ?

Reply:
It's correct, feel free to leave it as it is :-) -> https://developers.google.com/style/cross-references?hl=vi